src/py42/clients/alerts.py

```diff
@@ -136,3 +136,32 @@ def get_aggregate_data(self, alert_id):
             :class:`py42.response.Py42Response`
         """
         return self._alert_service.get_aggregate_data(alert_id)
+
+    def get_all_alert_details(self, query, ascending=True):
+        """
+        Helper method that combines `.get_all_pages()` and `.get_details()`.
+        Returns an iterator of alert detail objects in chronological order by alert
+        creation date.
```

Comment:
We could also note that sort-related parameters are ignored on the query, just in case the user is concerned about that for some reason

Reply:
Oh wait, I see a mention of it in the `ascending` doc

Reply:
Why not just support them? Could we do this if we let a user provide the `sort_key` as normal and use that in the `sorted` call?

Reply:
`createdAt` is still a good default for this though

Reply:
I suppose we could now that I think of it. As long as `sorted()` doesn't order things any differently than the server then it should just work I think...

Reply:
Updated to now just use whatever is on the query for sort_key/direction. We need to tweak the sort key param because the server takes capitalized strings but returns camel-case.